src/adapter/BUILD.bazel

```diff
@@ -24,7 +24,6 @@ rust_library(
     compile_data = [],
     crate_features = ["default"],
     data = [],
-    disable_pipelining = True,
```

Comment:
Did you mean to include this in this PR? It seems a bit unrelated.

Reply:
I didn't was rebased on the wrong branch by accident, removed!